proto/validator/accounts/v2/keymanager.proto

```diff
@@ -56,7 +56,7 @@ message SignRequest {
         ethereum.eth.v1alpha1.AttestationData attestation_data = 102;
         ethereum.eth.v1alpha1.AggregateAttestationAndProof aggregate_attestation_and_proof = 103;
         ethereum.eth.v1alpha1.VoluntaryExit exit = 104;
-        uint64 slot = 105;
+        uint64 slot = 105 [(gogoproto.casttype) = "github.com/prysmaticlabs/eth2-types.Slot"];
```

Comment:
Can this break upgrades if a validator is on a previous version ?

Reply:
I am not sure, but if it does -- we already breaking it, as the very next line reads: 
```golang
uint64 epoch = 106 [(gogoproto.casttype) = "github.com/prysmaticlabs/eth2-types.Epoch"];
```
I am not sure, however, how does it break anything -- internal type is the same, our code interfaces with some unsigned integer, type conversion only happens so that we can compile. Can you please elaborate how can I check if it breaks anything?

Reply:
Yeah from what I understand on protobufs, if we Marshal/Unmarshal it the same we *should* be ok. An easy way I guess would be to run on prymont, with your beacon node on the `eth2-types-slot` branch and your validator being on `develop`. If there are any issues it should be immediately obvious

Reply:
Still syncing on Pyrmont, will make sure I run validators on `develop` once it is synced, and let you know.